packages/lism-css/src/scss/trait/has/_transition.scss

```
@@ -2,7 +2,8 @@
   --duration: var(--transition-duration, 0.25s);
   --ease: ease;
   --delay: 0s;
-  --transitionProps: all;
+  // `all` は読み込み時やレイアウト変化時に意図しないアニメーションを起こすため、対象を見た目系プロパティに限定する。
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)